### ISSUE TITLE
Fuzzy misc enhancement requests

### ISSUE THREAD

**Issue by unkcpz** [aiidalab/aiidalab-qe#418]
I find some fuzzy UX problems during the use of the app, this issue is open as the centralized place to collect such problems:

- The workflow selection widget shows the description that the text is not clearly separated. 
- Start a new workflow and reset some widgets but not all. 
- Not super clear where to start a new workflow using workflow selector widget.
- Upload Structure widget keeps one increase in the counting when the new file is uploaded.

**Comment by unkcpz** [comment]
Thanks @AndresOrtegaGuerrero, I'll convert these requests to the issues so we don't forget to fix it.

> (problem - is a calculation so it will create a node when you call it , right? )

Yes, it will create nodes, but I won't worry about this duplicate node. 
Very good suggestion, though.

> Once you do a geometry optimization, it could be useful to show in the results the initial structure (with different coloring) vs final optimized, it could by overlapping?. Her comment is that there should be a way to do the visual comparison

Not sure there is an easy way to do this. It is not clear to me if the cell and atom positions are both changed, and how to align the structures to compare.

**Comment by superstar54** [comment]
> Not sure there is an easy way to do this. It is not clear to me if the cell and atom positions are both changed, and how to align the structures to compare.

An animation using the trajectory from the relaxing step is what we need.

**Comment by unkcpz** [comment]
> An animation using the trajectory from the relaxing step is what we need.

I don't think this is easy to do. First, is there already widget that supports showing the trajectory? Second, if it is an animation, which viewer angle should we use to show it? I guess you may have a similar component in BeautifulAtom @superstar54 ?

**Comment by superstar54** [comment]
`nglview` has an animation feature. We can test it. https://projects.volkamerlab.org/teachopencadd/talktorials/T017_advanced_nglview_usage.html
```
nv.show_asetraj(...)
```

**Comment by AndresOrtegaGuerrero** [comment]
> Not sure there is an easy way to do this. It is not clear to me if the cell and atom positions are both changed, and how to align the structures to compare.

Perhaps when the user does a cell optimization, the report should show the optimized geometry and a comparison with the initial cell parameter and the final cell parameters. For a geometry optimization we can just use coordinates of the initial atom as dummy atoms and color them all in blue (to overlap with the final structure), and just write a button that display the initial ones (or hide). The download one, should just download the final structure.

**Comment by superstar54** [comment]
> we actually have only two frames, the initial structure and the optimized one.

No, we can get the trajectory from the relax work chain.

> For a geometry optimization we can just use coordinates of the initial atom as dummy atoms and color them all in blue (to overlap with the final structure)

This only works if the initial structure and final structure are close. If some atoms move far away from their initial positions or even switch their positions (like in the adsorption, diffusion case, etc), the user will find it difficult to follow the atoms.

In my option, we should first try to use the animation. If it does not work, then we use this `dummy atoms` option.

**Comment by unkcpz** [comment]
Since critical issues are all solved I move this to next release and I also open an issue for comparing the structures as mentioned by @AndresOrtegaGuerrero .
